src/phoenix/db/models.py

```diff
@@ -51,6 +54,7 @@ class Trace(Base):
     project_rowid: Mapped[int] = mapped_column(ForeignKey("projects.id"))
     session_id: Mapped[Optional[str]]
     trace_id: Mapped[str]
+    # TODO(mikeldking): why is the start and end time necessary? just filtering?
```

Comment:
Yes, because there are moments in time where we can have multiple "root" spans (because the ultimate root has not arrived).